app/assets/v2/images/chains/tezos.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 25.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 193 237" style="enable-background:new 0 0 193 237;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#0D61FF;}
+	.st1{fill:#FFFFFF;}
+	.st2{fill:#262D33;}
+</style>
+<path class="st0" d="M175.8,171.2c-2.3-40.7-53.5-59.2-58.4-61.1c-0.2-0.1-0.2-0.3-0.1-0.5l52.8-53v-5.7c0-0.9-0.8-1.7-1.7-1.7H71.1
+	V17.3l0,0V7.7l-35.9,7.5v5.4h2c0,0,8.8,0,8.8,8.8v19.7H18.2c-0.5,0-1,0.5-1,1v12.5h28.9c0,0,0,13.3,0,28.8v39.1
+	c0,21.3,13.7,36.1,37.7,34.1c5.1-0.4,9.8-2.4,13.8-5c1.8-1.2,2.9-3.1,2.9-5.3v-6.7c-7.8,5.2-14.4,4.9-14.4,4.9
+	c-15.2,0-14.9-19.3-14.9-19.3V62.6h70l-50.4,50.8c-0.1,6.7-0.2,11.9-0.2,12c0,0.2,0.1,0.3,0.3,0.3c46.2,7.8,58.7,37.7,58.7,46.4
+	c0,0,5,42.3-37.3,45.2c0,0-27.7,1.2-32.6-9.9c-0.2-0.4,0-0.8,0.4-1c4.6-2.1,7.7-6.2,7.7-11.7c0-8.2-5-14.9-15.5-14.9
+	c-8.5,0-15.5,6.7-15.5,14.9c0,0-4,35.6,55.4,34.6C180,228.1,175.8,171.2,175.8,171.2z"/>
+</svg>
```

Comment:
^ do we wanna run svgo and compress the svg?

Reply:
Just not run it to all the project or will remove css on others svg files I recommend using https://jakearchibald.github.io/svgomg/ been using it for more than a year and the results are great